easyInterface/Diffraction/Interface.py

```diff
@@ -331,6 +331,8 @@ def updatePhases(self):
         Synchronise the phases in project dictionary by queering the calculator object.
         """
         phases = self.calculator.getPhases()
+        self._log.debug(phases)
+        self._log.debug(self.project_dict['phases'])
```

Comment:
Log should not be used this way. I'd suggest removal

Reply:
Debug logging is removed